packages/astro/src/core/build/static-build.ts

```diff
@@ -199,7 +200,7 @@ async function clientBuild(
 		build: {
 			...viteConfig.build,
 			emptyOutDir: false,
-			minify: 'esbuild',
+			minify,
```

Comment:
I think we could simply remove this line as minify with esbuild is Vite's default. And user config would be derived from `...viteConfig.build` above.